### PR TITLE
[bitnami/seaweedfs] Configure tests

### PR DESCRIPTION
**Pull request by fmulero**
### Description of the change

Initial set of tests for SeaweedFS container.

### Benefits

Ensure binary file is running and version is correct.

### Possible drawbacks

None

### CODE REVIEW

File: .vib/seaweedfs/vib-verify.json [hunk 1]
```diff
@@ -50,6 +50,21 @@
               "OS"
             ]
           }
+        },
+        {
+          "action_id": "goss",
```
Comment (andresbono) [inline .vib/seaweedfs/vib-verify.json:55]: The default entrypoint/cmd of the container keeps it up, right? No need for custom runtime parameters?

**Review by andresbono — DISMISSED** [review]
LGTM

File: .vib/seaweedfs/vib-verify.json (hunk 1 above)
Comment (fmulero) [reply .vib/seaweedfs/vib-verify.json:55]: Good catch! I forgot it

**Review by andresbono — APPROVED** [review]
LGTM

**Pull request merged**